packages/assistify-help-request/server/migrations.js

```diff
@@ -13,19 +13,38 @@ const removeObsoleteSubscriptions = function() {
 				'localField': 'u._id',
 				'foreignField': '_id',
 				'as': 'users'
-			}},
-		{ '$match': { 'users._id': { '$exists': false } } }
+			}
+		},
+		{'$match': {'users._id': {'$exists': false}}}
 	]).forEach(function(errSubscription) {
 		// we can bypass the cached collection as this will affect obsolete values
-		RocketChat.models.Subscriptions.model.remove({ _id: errSubscription._id.toString() });
+		RocketChat.models.Subscriptions.model.remove({_id: errSubscription._id.toString()});
 		counter++;
 	});
 	return counter;
 };
 
+function makePrivateRoomsSecret() {
+	let count = 0;
+	RocketChat.models.Rooms.findByType('p').forEach((room) => {
+		if ('customFields' in room) {
```

Comment:
I'd favor not to use custom fields but to enhance the model (e. g. with a `hidden` property)

Reply:
Extended the model schema with a new field `secret` instead of using the custom Fields.